thetower/dtower/tourney_results/data.py

```diff
@@ -167,7 +166,7 @@ def get_row_to_role(df: pd.DataFrame, league):
 
 def _load_tourney_results(
     result_files: list[tuple[str, str, list[BattleCondition]]],
-    league=champ,
+    league=legend,
```

Comment:
I'd leave this function as is. It's a legacy function that hopefully is not used in majority of the app anymore, and where it's used it may stay champ